frontend/src/metabase/core/components/EditableText/EditableText.tsx

```diff
@@ -17,6 +17,7 @@ export type EditableTextAttributes = Omit<
 export interface EditableTextProps extends EditableTextAttributes {
   initialValue?: string | null;
   placeholder?: string;
+  isOptional?: boolean;
```

Comment:
This is not the cleanest way to do it, but other options would require significant changes. What I've tried:
- Making this component controlled - would require wrapping it in another component every time it's used. Feels weird and doesn't bring any value.
- Making the query builder optimistically update the question state. It kind of works but causes some side-effects. I'm not confident enough that I can resolve all of them.

So adding a simple flag like this to avoid removing required properties, e.g. question titles, seems like a practical solution.